### DISCUSSION TITLE
Some way to contact server owners from the server menu.

### DISCUSSION THREAD

**Discussion by civilCornball** [FakeFishGames/Barotrauma#5988] [Ideas and suggestions]
Players often get banned due to a variety of reasons, most of them good.
This begs the question: What happens when they are banned for something they didn't do?

Currently, there is no real way to 'appeal' bans. You can't even attempt to contact the owner, since you don't know their SteamID!

As such, I would appreciate a way to contact the owner from the server menu. Even being able to access their profile would be enough!

**Comment by Cogs16** [comment]
I also want this alot, as a server owner I don't want to permanently ban people, however, I need a way to receive assurances or some sort of communication from them that will tell me that they either apologize or will attempt to behave better. I believe everyone deserves second chances, even on the internet.

**Reply by Veritius** [reply]
If you can figure out the owner, you can click on them in the player list and go to View Profile.

**Reply by civilCornball** [reply]
There's the thing: If you're banned, you can't connect to the server.
If you can't connect to the server, you can't view the player list by pressing TAB.
If you can't view the player list, you can't view the owner's profile.

it's a giant "for want of a nail" story
